### DISCUSSION TITLE
Newbie question on sticky content menu

### DISCUSSION THREAD

**Discussion by pranaykotas** [11ty/eleventy#3151] [Q&A]
I'm a newbie to Eleventy, trying to use this SSG for a non-profit think tank website. I want to know how does one enable the sticky left-side menu layout as is the case on this gov.uk [page](https://www.gov.uk/government/publications/national-semiconductor-strategy/national-semiconductor-strategy#our-plan-to-safeguard-the-uk-against-supply-chain-disruption-and-build-supply-chain-resilience)? 

I essentially want a markdown article sub-headers to appear on the left as a sticky menu so that a reader can navigate easily.

**Comment by btrem** [comment]
You can use [markdown-it anchor](https://www.npmjs.com/package/markdown-it-anchor) to automatically add id attributes to each heading. Then use [eleventy toc plugin](https://www.npmjs.com/package/eleventy-plugin-toc) which provides a filter to generate a toc. Finally, use css to place the toc menu on the left, fixed, or whatever you want. (I didn't look at the gov.uk page, but I imagine you don't need specific css help.)
